pyqtgraph/functions.py

```diff
@@ -1765,15 +1765,15 @@ def gaussianFilter(data, sigma):
     return filtered + baseline
     
     
-def downsample(data, n, axis=0, xvals='subsample'):
+def downsample(data, n, axis=0, xvals='subsample', *, nanPolicy='ignore'):
```

Comment:
renaming 'nan_policy' to 'nanPolicy' seems like a good idea for matching our Qt-inspired case style.
Shouldn't 'ignore' be 'omit' if we are aiming to resemble SciPy conventions?